package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "lint": "standard",
     "unit": "tap test.js",
-    "test": "npm run lint && npm run unit"
+    "test": "npm run lint && npm run unit",
+    "generate-readme": "sed -e '/\\${help.txt}/r help.txt' -e '/\\${help.txt}/d' README.tmpl  > README.md"
```

Comment:
Can you use node for this instead of sed? I would like to keep the env windows-friendly.

Reply:
Updated